salt/runners/jobs.py

```diff
@@ -200,6 +209,9 @@ def list_jobs(ext_source=None,
     .. code-block:: bash
 
         salt-run jobs.list_jobs
+        salt-run jobs.list_jobs search_function='test.*' search_target='localhost' search_metadata='{"bar": "foo"}'
+        salt-run jobs.list_jobs start_time='2015, Mar 16 19:00' end_time='2015, Mar 18 22:00'
```

Comment:
Don't have to be in that format, they can be any format supported by Dateutil.  I'll include a note.